neat_internal.h

```diff
@@ -30,6 +30,14 @@
 
 #define NEAT_MAX_NUM_PROTO 4
 
+#if defined(USRSCTP_SUPPORT) || defined(HAVE_NETINET_SCTP_H)
+#define SCTP_SUPPORTED
+#else
+#if defined(__linux__) // Need a further test for Linux SCTP support
+#define SCTP_SUPPORTED
```

Comment:
Just for the record: going with this suggestion, the SCTP_SUPPORTED constant has been removed. 

Running into some bugs in the kind of oddball configuration of Linux kernel SCTP enabled without having the proper headers, see also #86. My suggestion there is to say that it is a bad configuration and avoid using kernel SCTP at all in that case.